doc/swish/erlang.tex

```diff
@@ -1394,6 +1405,27 @@ \subsection{Miscellaneous}
 for the file \var{path} using \var{mode}, which defaults to
 \code{\#o777}. It returns \var{path}.
 
+% ----------------------------------------------------------------------------
+\defineentry{make-osi-input-port}
+\begin{procedure}
+  \code{(make-osi-input-port \var{p})}
+\end{procedure}
+\returns{} a binary input port
+
+The \code{make-osi-input-port} procedure returns a custom binary input
+port that reads from osi-port \var{p}. It does not track position.
+
```

Comment:
I like the idea of specifying the close behavior. Because not all osi-ports have underlying file descriptors, I'd prefer to mention that the underlying file descriptor is not closed in the documentation for open-fd-port.